pallets/fusion/src/types.rs

```diff
@@ -0,0 +1,368 @@
+use frame_support::pallet_prelude::*;
+use sp_core::H160;
+use sp_runtime::Perbill;
+use sp_staking::EraIndex;
+
+use crate::*;
+
+/// Type representing an EVM address
+pub type EvmAddress = H160;
+
+/// Type representing a balance for external currency
+pub type FusionCurrencyBalance = u128;
+
+/// Type to represent points
+pub type Points = u128;
+
+/// Type of the currency id
+pub type CurrencyId = u32;
+
+/// Type of the pool id
+pub type PoolId = u32;
+
+/// Prefix used for storing accounts
+pub const FUNDS_ACCOUNT_PREFIX: &str = "funds";
```

Comment:
In favour of using Enum for all AccountPrefixes, similar to how it's done in nom pools. Any specific advantage of using str here?

Reply:
[ref](https://github.com/availproject/polkadot-sdk/blob/09254eb9f8670f942c68914ff6fc0ffb503aaf23/substrate/frame/nomination-pools/src/lib.rs#L451)